package-lock.json

```diff
@@ -0,0 +1,2591 @@
+{
```

Comment:
I don't have a huge preference for this, but since this is new, I'd prefer to use a yarn lockfile, since it will auto fix merge conflicts and in general plays nicer with git.

Reply:
That's just a file rename right? Not totally sure how `yarn` plays with `npm`